client/src/components/backToAllReviews.jsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const BackToAllReviews = props => (
+  <div>hello</div>
```

Comment:
Assuming this will be replaced with further content later?

Reply:
```import React from 'react';
import PropTypes from 'prop-types';

const BackToAllReviews = ({ handleBacktoAllReviewsClick, currentReviewsLength, searchedWord }) => (
  <div>
    <div className="mentions">{currentReviewsLength} guests have mentioned "{searchedWord}"</div>
    <button className="backToAllReviews" onClick={handleBacktoAllReviewsClick}>Back to all reviews</button>
  </div>
);

export default BackToAllReviews;

BackToAllReviews.propTypes = {
  handleBacktoAllReviewsClick: PropTypes.func.isRequired,
  currentReviewsLength: PropTypes.number.isRequired,
  searchedWord: PropTypes.string.isRequired,
};```

This is what that component should look like. I think you are looking at an old commit somehow.